core/store/src/trie/mod.rs

```diff
@@ -1091,6 +1091,13 @@ impl Trie {
         Ok(())
     }
 
+    #[tracing::instrument(
+        level = "trace",
+        target = "store::trie",
+        "Trie::retrieve_raw_node",
+        skip_all,
+        fields(hash=%hash),
+    )]
```

Comment:
We might want to take this to `internal_retrieve_trie_node` as there are a couple of ways we can hit this function. `internal_retrieve_trie_node` is the main interaction layer with store.